com.github.CineEncoder.yaml

```diff
@@ -0,0 +1,11 @@
+name: Cine Encoder 2020 SE
```

Comment:
you need to declare the app-id in your manifest as well examples: https://github.com/flathub/org.kde.kdenlive/blob/master/org.kde.kdenlive.json#L2 

the absolutely minimal manifest requirements are in https://docs.flatpak.org/en/latest/first-build.html#add-a-manifest